### PR TITLE
'Bug 1793310': internal/scaffold: don't change /etc/passwd perms (#2469) [release-4.4]

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
Cherry-pick: https://github.com/operator-framework/operator-sdk/pull/2469/files

**Motivation**
https://bugzilla.redhat.com/show_bug.cgi?id=1793310
Bug 1793310

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed the scorecard basic test, `Writing into CRs has an effect`, to include the http.MethodPatch as part of its test criteria alongside http.MethodPut and http.MethodPost. ([#2509](https://github.com/operator-framework/operator-sdk/pull/2509))
+
```
Comment (joelanford) [inline CHANGELOG.md]: This is not the correct CHANGELOG entry for this bug.

**Comment by joelanford** [comment]
> Not mergeable. Merging is blocked by issue 25.

Also, it looks like this PR will need to target the `master` branch instead of `release-4.4` because we're still fast forwarding `release-4.4` from `master`.

File: CHANGELOG.md (hunk 1 above)
Comment (camilamacedo86) [reply CHANGELOG.md]: Ops. I did it pass. tks for the catch. I will fix.

**Comment by camilamacedo86** [comment]
Because of the review: I am closing this one and applying this fix in the master.

Screenshot 2020-02-13 at 23 27 21

**Pull request closed without merging**